src/labeler.ts

```diff
@@ -96,10 +97,21 @@ async function getLabelGlobs(
   client: ClientType,
   configurationPath: string
 ): Promise<Map<string, StringOrMatchConfig[]>> {
-  const configurationContent: string = await fetchContent(
-    client,
-    configurationPath
-  );
+  let configurationContent: string;
+
+  try {
+    configurationContent = (
+      await readFile(configurationPath, {
```

Comment:
Documentation on proper usage should be added.

Reply:
Sure. @lrstanley, could you also update README accordingly?